pkg/util/versions/versions_test.go

```diff
@@ -0,0 +1,36 @@
+package versions
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCalculateFCV(t *testing.T) {
+	assert.Equal(t, "4.2", CalculateFeatureCompatibilityVersion("4.2.0"))
```

Comment:
can we increase coverage by adding a test for a version < 3.4.0 and one that is not a valid semver version?

Reply:
will add some more tests to cover this!